appyters/miRNA_Target_Discovery/miRNA Target Discovery via CLIP-SEQ.ipynb

```diff
@@ -0,0 +1,2358 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%% appyter init\n",
+    "import json\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import pybedtools\n",
+    "import requests\n",
+    "import base64\n",
+    "import seaborn as sns\n",
+    "import matplotlib.pyplot as plt\n",
+    "import time\n",
+    "from urllib.parse import urlparse\n",
+    "from IPython.display import HTML, Markdown, display\n",
+    "\n",
+    "from matplotlib.ticker import MaxNLocator\n",
+    "\n",
+    "from appyter import magic\n",
+    "magic.init(lambda _=globals: _())"
```

Comment:
The first cell should be:
```python
#%%appyter init
from appyter import magic
magic.init(lambda _=globals: _())
```

Your other imports should be in the next cell. This cell is *removed* at runtime, have you tested this locally?

Reply:
yeah, it did work. hmm lemme fix it though